src/components/books/book.js

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable jsx-quotes */
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { removeBook, getAllBooks } from '../../redux/books/books';
+
+const book = () => {
+  const post = useSelector((state) => state.bookReducer);
+  const dispatch = useDispatch();
+  const handleClick = (id, e) => {
+    e.preventDefault();
+    dispatch(removeBook(id));
+  };
+  useEffect(() => {
+    dispatch(getAllBooks());
+  }, []);
+
+  return (
+    <>
+      {post.map((book) => (
+        <div
+          key={book.item_id}
+          className="row"
+        >
+          <section className="grid-item">
+            <div>
+              <p><span className="category">{book.category}</span></p>
+              <p><span className="title">{book.title}</span></p>
+              <p><span className="author">{book.author}</span></p>
+            </div>
+            <ul className="list">
+              <li>
+                <button type="button">Comment</button>
+              </li>
+              <li>
+                <button
+                  className='cursor-pointer'
+                  onClick={(e) => handleClick(book.item_id, e)}
+                  type='button'
+                >
+                  Remove
+                </button>
+              </li>
+              <li>
+                <button type="button">Edit</button>
+              </li>
+            </ul>
+          </section>
```

Comment:
- [ ] Great job so far, because the design has to look exactly like the one provided, please remove the border before the completion status and also the border before the chapter should go all the way. Kindly fix it.

### Your Implementation
![image](https://user-images.githubusercontent.com/59416136/168379512-9eb809ff-6b7f-4cd2-b200-03bed84f6a93.png)

### Zephin Design

![image](https://user-images.githubusercontent.com/59416136/168380025-fc10c67e-cec8-4982-93dd-5425ad954b78.png)